integration_tests/run_pyspark_from_build.sh

```diff
@@ -28,9 +28,11 @@ else
     # support alternate local jars NOT building from the source code
     if [ -d "$LOCAL_JAR_PATH" ]; then
         CUDF_JARS=$(echo "$LOCAL_JAR_PATH"/cudf-*.jar)
-        PLUGIN_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark*.jar)
+        PLUGIN_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark_*.jar)
         TEST_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark-integration-tests*.jar)
         UDF_EXAMPLE_JARS=$(echo "$LOCAL_JAR_PATH"/rapids-4-spark-udf-examples*.jar)
+        ## reset SCRIPTPATH to support alternate scripts path on the cloud environment
+        SCRIPTPATH="$LOCAL_JAR_PATH/integration_tests"
```

Comment:
reset SCRIPTPATH to support alternate scripts path on the cloud environment

Reply:
But it is not the SCRIPTPATH any longer.  SCRIPTPATH is where this script resides and we are finding other things relative to that path.  I would prefer to have a new variable like INTEGRATION_TEST_ROOT or something that can be set to SCRIPTPATH in the common case, and then set here to what we want it to be.